src/core/tokens.ts

```diff
@@ -37,27 +37,6 @@ export class TokenTransfer {
     readonly token: Token;
     readonly amount: bigint;
 
-    /**
-     * @deprecated field. Use "token.identifier" instead.
-     */
-    readonly tokenIdentifier: string;
-
-    /**
-     * @deprecated field. Use "token.nonce" instead.
-     */
-    readonly nonce: number;
-
-    /**
-     * @deprecated field. Use "amount" instead.
-     */
-    readonly amountAsBigInteger: BigNumber;
-
-    /**
-     * @deprecated field. The number of decimals is not a concern of "sdk-core".
-     * For formatting and parsing amounts, use "sdk-dapp" or "bignumber.js" directly.
-     */
-    readonly numDecimals: number;
-
     constructor(options: { token: Token; amount: bigint } | ILegacyTokenTransferOptions) {
```

Comment:
Is it worth to still have the legacy flow?

Reply:
removed